contributions/api/utils/adminutils.py

```diff
@@ -49,3 +53,36 @@ def check_if_reviewer(login_id):
             return True
 
     return False
+
+def send_email(mail_to, subject, message):
+    """
+    Method to send email to a user and print if success or failure
+    Args:
+        mail_to (str): email id of recipient
+        subject (str): subject of the email
+        message (str): message of the email
+    Returns:
+        (bool): True if success, False if failure
+        (str): Error code is failure, empty string if success
+        (str): Error message if failure, empty string if success
+    """
+    mail_subject = subject
+    mail_body = message
+
+    mail_from = cfg.SENDER_EMAIL
+    password = cfg.SENDER_EMAIL_PASSWORD
+
+    mimemsg = MIMEMultipart()
+    mimemsg['From'] = mail_from
+    mimemsg['To'] = mail_to
+    mimemsg['Subject'] = mail_subject
+    mimemsg.attach(MIMEText(mail_body, 'plain'))
+    try:
+        connection = smtplib.SMTP(host=cfg.SMTP_HOST, port=cfg.SMTP_PORT)
+        connection.starttls()
+        connection.login(mail_from, password)
+        connection.send_message(mimemsg)
+        connection.quit()
+        return True, ' ', ' '
+    except smtplib.SMTPException as stmpex:
```

Comment:
A couple of comments related to exception handling. First, it needs to be something like below:

```
except smtplib.SMTPException as smtp_ex:
        return False, smtp_ex.strerror, smtp_ex.errno
```

Please test with another incorrect username and password to ensure that the error message and code can be logged properly. 

Also, I feel that if there is an exception in sending the email notification, it just needs to be logged as an error. The response to the client can still be HTTP 200 because the POST (creating the record) would have actually succeeded by then. What do you think?

Reply:
I.e, I was thinking sending the notification is an optional component, and it should probably not affect the response of the POST endpoint for now. When we have a new endpoint for sending email notifications, it can obviously have specific error codes.